amplify/__tests__/functions/send-email/handler.test.ts

```diff
@@ -2,6 +2,7 @@ import { mockDeep } from 'jest-mock-extended';
 import { SendRawEmailCommandInput, SESClient } from '@aws-sdk/client-ses';
 import { handler } from '../../../functions/send-email/handler';
 import type { Schema } from '../../../data/resource';
+import { emailTemplate } from '../../../functions/send-email/email-template';
```

Comment:
Could make an index.ts file and export everything from /functions/send-email to save on having to double import?